javascript/sdk/lib/controllers/build.ts

```diff
@@ -0,0 +1,83 @@
+import Client from "../client";
```

Comment:
I added a few 'purge' routes on the API that we'll need on the FE; might as well add here anytime.